microsoft/testsuites/core/timesync.py

```diff
@@ -199,7 +209,11 @@ def timesync_check_unbind_clocksource(self, node: Node) -> None:
     def timesync_check_unbind_clockevent(self, node: Node) -> None:
         if node.shell.exists(PurePosixPath(self.current_clockevent)):
             # 1. Current clock event name is 'Hyper-V clockevent'.
-            clock_event_name = "Hyper-V clockevent"
+            lscpu = node.tools[Lscpu]
+            if "x86_64" == lscpu.get_architecture():
+                clock_event_name = "Hyper-V clockevent"
+            elif "aarch64" == lscpu.get_architecture():
+                clock_event_name = "arch_sys_timer"
```

Comment:
Add `else` to raise exception on not recognized string.

Reply:
Done

Reply:
Done